### DISCUSSION TITLE
No "new project" button in mobile view for projects beta

### DISCUSSION THREAD

**Discussion by the-moisrex** [community/community#6910] [Projects and Issues] [answered]
Unfortunately there's no "new project" button in beta version of projects when mobile device is being used. I have to enable desktop view, click on the `new project (beta)` and the  switch back to mobile view for that.

**Comment by aconsuegra** [comment, marked as answer]
Hi @the-moisrex 
thanks for reaching out. Are you using the GitHub mobile native apps or the mobile website of github.com? Projects (old or beta versions) are not supported in the native mobile apps yet.

**Reply by the-moisrex** [reply]
> Hi @the-moisrex 
> thanks for reaching out. Are you using the GitHub mobile native apps or the mobile website of github.com? Projects (old or beta versions) are not supported in the native mobile apps yet.

I'm of course using the mobile view of the website. I'm aware that the native app doesn't support this feature yet. The native app doesn't support lots of more features that the web app does! But here I'm just notifying that the web app's `new project` button just doesn't show up anymore in the projects beta version's page so the only way to create a new beta project in mobile web app is to enable desktop mode.

**Reply by aconsuegra** [reply]
Thanks for your reply! The `📱 Mobile Feedback` category is meant to be used to provide feedback about the native mobile apps, hence my question.

For feedback about the new Projects, you can start a discussion in https://github.com/github/feedback/discussions/categories/issues-feedback

**Reply by JannesPeters** [reply]
I will create a bug for the projects beta (web) team.
